gensrc/proto/runtime_profile.proto

```diff
@@ -0,0 +1,78 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
```

Comment:
do not add such structure, just use profile's thrift structure and serialize it to bytes and send the binary data with protobuf message.

Reply:
done